lib/awsapi/config.go

```diff
@@ -206,6 +206,9 @@ func (cfg *Config) getAPICredentials() (*credentials, error) {
 		AccessKeyID:     cfg.defaultAccessKey,
 		SecretAccessKey: cfg.defaultSecretKey,
 	}
+	if len(acNew.AccessKeyID) > 0 && len(acNew.SecretAccessKey) > 0 {
```

Comment:
This path ignores roleArn and it'll make impossible to use it with static credentials.

Reply:
updated code and added tests